package.json

```diff
@@ -2,7 +2,8 @@
   "name": "react-waypoint",
   "version": "7.2.0",
   "description": "A React component to execute a function whenever you scroll to an element.",
-  "main": "build/waypoint.js",
+  "main": "build/index.js",
```

Comment:
if both of these are just "build" - which is the best practice - does that mean rollup can't figure out the destination filenames?

Reply:
I don't understand your question. What is a best practice? What is rollup not figuring out?

Reply:
`"main": "build", "module": "build"`.

I don't see anywhere else `index.js` or `index.mjs` are specified tho, which leads me to believe that rollup infers the filenames from the package.json fields.

Reply:
No, it doesn't infer anything. It is configured with an explicit dependency on these fields in the rollup config. https://github.com/brigade/react-waypoint/pull/220/files#diff-ff6e5f22a9c7e66987b19c0199636480

```diff
+    targets: [
+      { dest: pkg.main, format: 'cjs' },
+      { dest: pkg.module, format: 'es' }
+    ],
```

Reply:
That specifies the format, but not the file extension. Where would spaghetti and meatballs go?

Reply:
It specifies the path currently. If we change it to spaghetti and meatballs, it would go in the package.json.

Reply:
Right - that confirms my initial hunch, which is that putting just `"build"` wouldn't work, because it would deprive rollup of the actual path.